Source/ACE/Entity/WorldObject.cs

```diff
@@ -414,92 +442,92 @@ public float Workmanship
         /// </summary>
         public uint? SetupTableId
         {
-            get { return AceObject.SetupTableId; }
-            set { AceObject.SetupTableId = value; }
+            get { return AceObject.GetDataIdProperty(PropertyDataId.Setup); }
+            set { AceObject.SetDataIdProperty(PropertyDataId.Setup, value); }
         }
 
         public PhysicsDescriptionFlag PhysicsDescriptionFlag { get; set; }
 
         public PhysicsState PhysicsState
         {
-            get { return (PhysicsState)AceObject.PhysicsState; }
-            set { AceObject.PhysicsState = (uint)value; }
+            get { return (PhysicsState)AceObject.GetIntProperty(PropertyInt.PhysicsState); }
+            set { AceObject.SetIntProperty(PropertyInt.PhysicsState, (uint)value); }
         }
 
         /// <summary>
         /// mtable_id in aclogviewer This is the sound table for the object.   Looked up from dat file.
         /// </summary>
         public uint? MotionTableId
         {
-            get { return AceObject.MotionTableId; }
-            set { AceObject.MotionTableId = value; }
+            get { return AceObject.GetDataIdProperty(PropertyDataId.MotionTable); }
+            set { AceObject.SetDataIdProperty(PropertyDataId.MotionTable, value); }
         }
         /// <summary>
         /// stable_id in aclogviewer This is the sound table for the object.   Looked up from dat file.
         /// </summary>
         public uint? SoundTableId
         {
-            get { return AceObject.SoundTableId; }
-            set { AceObject.SoundTableId = value; }
+            get { return AceObject.GetDataIdProperty(PropertyDataId.SoundTable); }
+            set { AceObject.SetDataIdProperty(PropertyDataId.SoundTable, value); }
         }
         /// <summary>
         /// phstable_id in aclogviewer This is the physics table for the object.   Looked up from dat file.
         /// </summary>
         public uint? PhysicsTableId
         {
-            get { return AceObject.PhysicsTableId; }
-            set { AceObject.PhysicsTableId = value; }
+            get { return AceObject.GetDataIdProperty(PropertyDataId.PhysicsEffectTable); }
+            set { AceObject.SetDataIdProperty(PropertyDataId.PhysicsEffectTable, value); }
         }
 
         /// <summary>
         /// This is used for equiped items that are selectable.   Weapons or shields only.   Max 2
         /// </summary>
         public uint? Parent
```

Comment:
I realize this isn't your change, but similar to Container, this should be ParentId.